core/src/com/unciv/models/translations/TranslationFileWriter.kt

```diff
@@ -206,6 +206,7 @@ object TranslationFileWriter {
             fun submitString(item: Any) {
                 val string = item.toString()
 
+                //todo This needs to learn about {}-using "AND" filters
```

Comment:
We could then use the same splitFilter function in the translationFileWriter, if the parameter is a multi-filter, split and take the first. Should be good enough, I think.